amp/repository/aim/view/auditLoggerManager.jsp

```diff
@@ -75,6 +84,17 @@ function toggleLoggs(){
 	document.aimAuditLoggerManagerForm.submit();
 }
 
+function dateFilter(){

```

Comment:
I would just show the date filter to avoid making the user click another check to show the date